docs-new/docs/graphql/core/api-reference/syntax-defs.mdx

```diff
@@ -241,229 +213,204 @@ String
 
-## SelectPermission
+## SelectPermission {#selectpermission}
 
 | Key                | Required | Schema                      | Description                                                        |
 |--------------------|----------|-----------------------------|--------------------------------------------------------------------|
-| columns            | true     | `PGColumn` array (or) `'*'` | Only these columns are selectable (or all when `'*'` is specified) |
-| computed_fields    | false    | `ComputedFieldName` array   | Only these computed fields are selectable                          |
-| filter             | true     | `BoolExp`                   | Only the rows where this expression holds true are selectable      |
+| columns            | true     | [PGColumn](#pgcolumn) array (or) `'*'` | Only these columns are selectable (or all when `'*'` is specified) |
+| computed_fields    | false    | [ComputedFieldName](#computedfieldname) array   | Only these computed fields are selectable                          |
+| filter             | true     | [BoolExp](#boolexp)                   | Only the rows where this expression holds true are selectable      |
 | limit              | false    | `Integer`                   | The maximum number of rows that can be returned                    |
 | allow_aggregations | false    | `Boolean`                   | Toggle allowing aggregate queries                                  |
 
-## UpdatePermission
+## UpdatePermission {#updatepermission}
 
 | Key     | Required | Schema                      | Description                                                                            |
 |---------|----------|-----------------------------|----------------------------------------------------------------------------------------|
-| columns | true     | `PGColumn` array (or) `'*'` | Only these columns are selectable (or all when `'*'` is specified)                     |
-| filter  | true     | `BoolExp`                   | Only the rows where this precondition holds true are updatable                         |
-| check   | false    | `BoolExp`                   | Postcondition which must be satisfied by rows which have been updated                  |
-| set     | false    | `ColumnPresetExp`           | Preset values for columns that can be sourced from session variables or static values. |
+| columns | true     | [PGColumn](#pgcolumn) array (or) `'*'` | Only these columns are selectable (or all when `'*'` is specified)                     |
+| filter  | true     | [BoolExp](#boolexp)                   | Only the rows where this precondition holds true are updatable                         |
+| check   | false    | [BoolExp](#boolexp)                   | Postcondition which must be satisfied by rows which have been updated                  |
+| set     | false    | [ColumnPresetExp](#columnpresetexp)           | Preset values for columns that can be sourced from session variables or static values. |
```

Comment:
both id and label mismatch, missed `s`, ColumnPreset`s`

`[ColumnPresetsExp](#columnpresetsexp)`

Reply:
Label is ColumnPresetsExp and id is #columnpresetexp in rst file :(